sci-mathematics/cipi/cipi-1.0-r1.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="Computing information projections iteratively"
+HOMEPAGE="https://github.com/tom111/cipi"
+SRC_URI="https://github.com/tom111/cipi/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc"
+
+DEPEND="
+	dev-libs/boost
```

```suggestion
	dev-libs/boost:=
```
Probably it should be rebuilt on boost slot changes